### ISSUE TITLE
v1.6.0-rc.0 Docker container won't start

### ISSUE THREAD

**Issue by sanders41** [meilisearch/meilisearch#4267]
**Describe the bug**
The v1.6.0-rc.0 Docker image is erroring when starting.

**To Reproduce**
1. Download the v1.6.0-rc.0 docker images. `docker pull getmeili/meilisearch:v1.6.0-rc.0`
2. Run the image `docker run --rm -p 7700:7700 getmeili/meilisearch:v1.6.0-rc.0 meilisearch --master-key=masterKey --no-analytics`
3. An error is thrown

```console
Error loading shared library libstdc++.so.6: No such file or directory (needed by /bin/meilisearch)
Error relocating /bin/meilisearch: _ZNSt8ios_base4InitC1Ev: symbol not found
Error relocating /bin/meilisearch: _ZNSt8ios_base4InitD1Ev: symbol not found
```

**Expected behavior**
Image should start without error.

**Screenshots**

**Meilisearch version:**
v1.6.0-rc.0

**Additional context**
Additional information that may be relevant to the issue.
[e.g. architecture, device, OS, browser]

**Comment by curquiza** [comment]
Thank you very much @sanders41, definitely, you saved this release 👍 
I have the same issue on my side, and thanks for the PR! taking a look at it

**Comment by curquiza** [comment]
Closed by #4268 

Will be released in rc1
